src/stories/layout-helpers/StyledHorizontalStack.tsx

```diff
@@ -1,17 +1,33 @@
 import { PropsWithChildren } from 'react';
+import { convertToRem } from './LayoutFunctions';
 
 export interface StyledHorizontalStackProps extends PropsWithChildren {
   gap?: number;
   spanAcross?: boolean;
+  marginX?: number;
+  marginY?: number;
 }
 
-export default function StyledHorizontalStack({ children, gap = 0, spanAcross }: StyledHorizontalStackProps) {
-  const gapRem = gap / 4;
-  const spacing = gapRem + 'rem';
+export default function StyledHorizontalStack({
+  children,
+  gap = 0,
+  spanAcross,
+  marginY,
+  marginX,
+}: StyledHorizontalStackProps) {
+  let mY: string | undefined;
+  let mX: string | undefined;
+
+  const spacing = convertToRem(gap);
+  marginY !== undefined ? (mY = convertToRem(marginY)) : (mY = '0');
+  marginX !== undefined ? (mX = convertToRem(marginX)) : (mX = '0');
```

Comment:
```ts
  const spacing = convertToRem(gap);
  const mY = marginY != undefined ? convertToRem(marginY) : '0';
  const mX = marginX != undefined ? convertToRem(marginX) : '0';
```

Reply:
Only use `!=` and not `!==`, if you compare something to `undefined`. The `!=` also catches `null` values.